File: src/test/java/com/codingbat/warmup1/NearHundredTest.java
package com.codingbat.warmup1;

import org.junit.Test;

import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;

public class NearHundredTest {
  @Test
  public void expect93ToBeNearHundred() {
    assertTrue(NearHundred.nearHundred(93));
  }

  @Test
  public void expect90ToBeNearHundred() {
    assertTrue(NearHundred.nearHundred(90));
  }

  @Test
  public void expect89ToNotBeNearHundred() {
    assertFalse(NearHundred.nearHundred(89));
  }

  @Test
  public void expect110ToBeNearHundred() {
    assertTrue(NearHundred.nearHundred(110));
  }

  @Test
  public void expect111ToNotBeNearHundred() {
    assertFalse(NearHundred.nearHundred(111));
  }

  @Test
  public void expect121ToNotBeNearHundred() {
    assertFalse(NearHundred.nearHundred(121));
  }

  @Test
  public void expectZeroToNotBeNearHundred() {
    assertFalse(NearHundred.nearHundred(0));
  }

  @Test
  public void expectFiveToNotBeNearHundred() {
    assertFalse(NearHundred.nearHundred(5));
  }

  @Test
  public void expect191ToBeNearHundred() {
    assertTrue(NearHundred.nearHundred(191));
  }

  @Test
  public void expect189ToNotBeNearHundred() {
    assertFalse(NearHundred.nearHundred(189));
  }

  @Test
  public void expect190ToBeNearHundred() {
    assertTrue(NearHundred.nearHundred(190));
  }

  @Test
  public void expect200ToBeNearHundred() {
    assertTrue(NearHundred.nearHundred(200));
  }

  @Test
  public void expect210ToBeNearHundred() {
    assertTrue(NearHundred.nearHundred(210));
  }

  @Test
  public void expect211ToNotBeNearHundred() {
    assertFalse(NearHundred.nearHundred(211));
  }

  @Test
  public void expect290ToNotBeNearHundred() {
    assertFalse(NearHundred.nearHundred(290));
  }
}

File: src/test/kotlin/com/codingbat/warmup1/NearHundredTest.kt
package com.codingbat.warmup1

import org.junit.Test
import kotlin.test.assertFalse
import kotlin.test.assertTrue

class NearHundredTestKt {
  @Test
  fun expect93ToBeNearHundred() {
    assertTrue(nearHundred(93))
  }

  @Test
  fun expect90ToBeNearHundred() {
    assertTrue(nearHundred(90))
  }

  @Test
  fun expect89ToNotBeNearHundred() {
    assertFalse(nearHundred(89))
  }

  @Test
  fun expect110ToBeNearHundred() {
    assertTrue(nearHundred(110))
  }

  @Test
  fun expect111ToNotBeNearHundred() {
    assertFalse(nearHundred(111))
  }

  @Test
  fun expect121ToNotBeNearHundred() {
    assertFalse(nearHundred(121))
  }

  @Test
  fun expectZeroToNotBeNearHundred() {
    assertFalse(nearHundred(0))
  }

  @Test
  fun expectFiveToNotBeNearHundred() {
    assertFalse(nearHundred(5))
  }

  @Test
  fun expect191ToBeNearHundred() {
    assertTrue(nearHundred(191))
  }

  @Test
  fun expect189ToNotBeNearHundred() {
    assertFalse(nearHundred(189))
  }

  @Test
  fun expect190ToBeNearHundred() {
    assertTrue(nearHundred(190))
  }

  @Test
  fun expect200ToBeNearHundred() {
    assertTrue(nearHundred(200))
  }

  @Test
  fun expect210ToBeNearHundred() {
    assertTrue(nearHundred(210))
  }

  @Test
  fun expect211ToNotBeNearHundred() {
    assertFalse(nearHundred(211))
  }

  @Test
  fun expect290ToNotBeNearHundred() {
    assertFalse(nearHundred(290))
  }
}
